error: ‘DBUS_MESSAGE_ITER_INIT_CLOSED’ was not declared in this scope

```
$ cat /etc/redhat-release 
Red Hat Enterprise Linux Workstation release 7.7 (Maipo)
$ cmake .. && make
-- [INFO] Linux Host Detected
-- The C compiler identification is GNU 7.3.1
-- The CXX compiler identification is GNU 7.3.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /opt/rh/devtoolset-7/root/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /opt/rh/devtoolset-7/root/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PkgConfig: /bin/pkg-config (found version "0.27.1") 
-- Checking for one of the modules 'dbus-1'
-- Using DBUS from path: /usr/include/dbus-1.0;/usr/lib64/dbus-1.0/include
-- [INFO] Building Native BLE
-- [INFO] Building Native BLE C Wrapper
-- [INFO] Building Native BLE Test Application
-- Configuring done
-- Generating done
-- Build files have been written to: /home/user/src/nativeble/build
Scanning dependencies of target nativeble-static
[  2%] Building CXX object CMakeFiles/nativeble-static.dir/src-ble/NativeBleController.cpp.o
[  4%] Building CXX object CMakeFiles/nativeble-static.dir/src-ble/linux/NativeBleInternal.cpp.o
[  6%] Building CXX object CMakeFiles/nativeble-static.dir/src-ble/linux/bluezdbus/BluezAdapter.cpp.o
[  9%] Building CXX object CMakeFiles/nativeble-static.dir/src-ble/linux/bluezdbus/BluezAgent.cpp.o
[ 11%] Building CXX object CMakeFiles/nativeble-static.dir/src-ble/linux/bluezdbus/BluezDevice.cpp.o
[ 13%] Building CXX object CMakeFiles/nativeble-static.dir/src-ble/linux/bluezdbus/BluezGattCharacteristic.cpp.o
[ 15%] Building CXX object CMakeFiles/nativeble-static.dir/src-ble/linux/bluezdbus/BluezGattService.cpp.o
[ 18%] Building CXX object CMakeFiles/nativeble-static.dir/src-ble/linux/bluezdbus/BluezService.cpp.o
[ 20%] Building CXX object CMakeFiles/nativeble-static.dir/src-ble/linux/bluezdbus/interfaces/Adapter1.cpp.o
[ 22%] Building CXX object CMakeFiles/nativeble-static.dir/src-ble/linux/bluezdbus/interfaces/Device1.cpp.o
[ 25%] Building CXX object CMakeFiles/nativeble-static.dir/src-ble/linux/bluezdbus/interfaces/GattCharacteristic1.cpp.o
[ 27%] Building CXX object CMakeFiles/nativeble-static.dir/src-ble/linux/bluezdbus/interfaces/GattService1.cpp.o
[ 29%] Building CXX object CMakeFiles/nativeble-static.dir/src-ble/linux/simpledbus/base/Connection.cpp.o
[ 31%] Building CXX object CMakeFiles/nativeble-static.dir/src-ble/linux/simpledbus/base/Holder.cpp.o
[ 34%] Building CXX object CMakeFiles/nativeble-static.dir/src-ble/linux/simpledbus/base/Logger.cpp.o
[ 36%] Building CXX object CMakeFiles/nativeble-static.dir/src-ble/linux/simpledbus/base/Message.cpp.o
/home/user/src/nativeble/src-ble/linux/simpledbus/base/Message.cpp: In member function ‘void SimpleDBus::Message::_invalidate()’:
/home/user/src/nativeble/src-ble/linux/simpledbus/base/Message.cpp:72:19: error: ‘DBUS_MESSAGE_ITER_INIT_CLOSED’ was not declared in this scope
     this->_iter = DBUS_MESSAGE_ITER_INIT_CLOSED;
                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated due to -Wfatal-errors.
make[2]: *** [CMakeFiles/nativeble-static.dir/build.make:277: CMakeFiles/nativeble-static.dir/src-ble/linux/simpledbus/base/Message.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:182: CMakeFiles/nativeble-static.dir/all] Error 2
make: *** [Makefile:103: all] Error 2
```

EDITS:
I'm new to dbus.  Here's the commit that introduced this API change: https://gitlab.freedesktop.org/dbus/dbus/-/commit/205fc2847dd2347733dbea74452b5cc11d937c14

--- a/src-ble/linux/simpledbus/base/Message.cpp
+++ b/src-ble/linux/simpledbus/base/Message.cpp
@@ -69,7 +69,11 @@ void Message::_invalidate() {
     this->_iter_initialized = false;
     this->_is_extracted = false;
     this->_extracted = Holder();
+    #ifdef DBUS_MESSAGE_ITER_INIT_CLOSED
     this->_iter = DBUS_MESSAGE_ITER_INIT_CLOSED;
+    #else
+    this->_iter = DBusMessageIter();
+    #endif
 }
 
 void Message::_safe_delete() {

